plan/rule_aggregation_push_down.go

```diff
@@ -185,7 +185,8 @@ func (a *aggregationOptimizer) decompose(aggFunc *aggregation.AggFuncDesc, schem
 	return result, schema
 }
 
-func (a *aggregationOptimizer) allFirstRow(aggFuncs []*aggregation.AggFuncDesc) bool {
+// IsAllFirstRow checks whether functions in `aggFuncs` are all FirstRow.
```

Comment:
Since it's exported and no longer a member method of `aggregationOptimizer`, it should be something in `expression/aggregation` package. Instead of the `plan` package.